packages/ui/src/styles/globals.css

```
@@ -258,6 +264,189 @@
+      color-mix(in srgb, var(--surface-frame-shadow-color) 1%, transparent);
+
+  box-shadow:
+    inset 0 var(--surface-frame-bevel-y) 0 -0.5px
+      var(--surface-frame-bevel-color),
+    var(--surface-frame-elevation),
+    0 0 0 var(--surface-frame-halo-gap) var(--surface-frame-halo-spacer),
+    0 0 0 var(--surface-frame-halo-spread) var(--surface-frame-halo-color),
+    0 0 0 var(--surface-frame-ring-spread) var(--surface-frame-ring-color) !important;
+}
+
+@utility surface-frame-elevation-none {
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)